honestcash-v2/src/environments/environment.ts

```diff
@@ -14,7 +14,7 @@ export const ENVIRONMENT_DEFAULTS: Environment = {
   env: 'local',
   apiUrl: 'http://localhost:8080/api', // https://honest.cash/api
   clientUrl: 'http://localhost:3010/', // https://honest.cash/
-  production: false
+  production: true
```

Comment:
this should not be changed. we have different env files.

Reply:
https://github.com/honest-cash/honestcash/blob/4cd563e76bb3465a1503e6cfa47fb5848eccb2c2/honestcash-v2/src/app/shared/guards/unauthorized.guard.ts#L37

I do this check while developing locally otherwise it keeps redirecting me to / of v2 not / of v1 so if you want `production: false` to remain I need to remove this and find another way to make my way while developing v2 (it really gets annoying the redirection and infinite loops). I mean production: true does not make any different whatsoever in the app neither I can access cli args with --env=production so if it does not hurt, lets keep it production true always and I turn it to false on local deveopment